hazelcast/src/test/java/com/hazelcast/config/YamlConfigBuilderTest.java

```diff
@@ -3458,6 +3458,23 @@ public void testMetricsConfig() {
         assertEquals(11, metricsMcConfig.getRetentionSeconds());
     }
 
+    @Override
+    @Test
+    public void testInstanceTrackingConfig() {
+        String yaml = ""
+                + "hazelcast:\n"
+                + "  instance-tracking:\n"
+                + "    enabled: false\n"
```

Comment:
Since it is disabled by default, I would test with `true`.

Reply:
Changed to `true`. Actually I intentionally set it to `false` because I was afraid it would create files that weren't being cleaned up. But hopefully writing to `/dummy/file` will fail :) If you have some ideas for file names that will definitely fail and throw an exception, we can change it to that.

Reply:
I think it's fine here, we don't start an instance with the created config, we test only the parser.

Reply:
Ah. I changed it on all places where we were testing some config, including where we start an instance (I think Spring?) so maybe I can revert there.